src/front-end/src/components/MenteeForm.js

```diff
@@ -2,23 +2,54 @@ import React, { Component } from 'react';
 import { Form, Button, Input } from 'antd';
 import TimezonePicker from 'react-bootstrap-timezone-picker';
 import 'react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css';
-import { marshallMenteeInfo, pushToDB } from '../helper-methods';
+import emailjs from 'emailjs-com';
+// import { marshallMenteeInfo, pushToDB } from '../helper-methods';
 import MenteeCompletion from './MenteeCompletion';
 
 class MenteeForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
       isSubmitted: false,
+      email: '',
+      name: '',
     };
     this.onFinish = this.onFinish.bind(this);
     this.onFinishFailed = this.onFinishFailed.bind(this);
+    this.sendEmail = this.sendEmail.bind(this);
+    this.handleEmailChange = this.handleEmailChange.bind(this);
+    this.handleNameChange = this.handleNameChange.bind(this);
+  }
+
+  handleEmailChange(event) {
+    this.setState({ email: event.target.value });
+  }
+
+  handleNameChange(event) {
+    this.setState({ name: event.target.value });
+  }
+
+  sendEmail() {
+    const email = this.state.email;
```

Comment:
When I run the code in VSCode, I get the same Error that Github is displaying on this PR ('Use object destructuring'). If I hover my mouse above the error line, and go to Quick Fix then click 'fix all auto-fixable problems' it seems to work. I'm not sure if this is the best way to solve the error though

Reply:
I disabled that rule in the linter's config file.

Reply:
> Great work the email's format is cool!
> Can we slightly modify what the email says, so that the 's' in MentorShip is lowercase, and instead of 'Welcome to be a mentee', have the email say 'Welcome to MentorMatch!'?

modified. Thanks for the feedback.